src/plugins/intel_npu/src/plugin/npuw/serialization.cpp

```diff
@@ -74,6 +74,12 @@ void ov::npuw::s11n::write(std::ostream& stream, const ov::Output<const ov::Node
     write(stream, var.get_names());
 }
 
+void ov::npuw::s11n::write(std::ostream& stream, const ov::Any& var) {
+    // FIXME: figure out a proper way to serialize Any (for config)
+    NPUW_ASSERT(var.is<std::string>() && "Can't serialize ov::Any which is not std::string!");
+    write(stream, var.as<std::string>());
```

Comment:
would it cause problems with serializing NPU_DPU_GROUPS if it is set as `4`, not `"4"`?

Reply:
I could handle `int` as well

Reply:
No need. I tested - it can be accessed as any int type